src/main/java/se/sundsvall/emailreader/api/model/Email.java

```diff
@@ -0,0 +1,7 @@
+package se.sundsvall.emailreader.api.model;
+
+import lombok.Builder;
+
+@Builder(setterPrefix = "with")
+public record Email(String to, String from, String subject, String message, String messageID) {
```

Comment:
messageId -> id ?

1: id is usually the name of the identificator in our API:s. 
2: message in messageID seems redundant, since this is a model of an emailMessage?